### ISSUE TITLE
truffle: command not found

### ISSUE THREAD

**Issue by Koldus** [trufflesuite/truffle#1027]
When installing truffle globally, truffle installation is completed but doesn't actually get installed in the folder with global packages.

```
/home/ec2-user/.nvm/versions/node/v6.14.3/lib/node_modules/node/bin/truffle -> /home/ec2-user/.nvm/versions/node/v6.14.3/lib/node_modules/node/lib/node_modules/truffle/build/cli.bundled.js
+ truffle@4.1.11
updated 1 package in 2.46s
```

Following which 'truffle init' always returns command not found error. I tried this locally or on EC2 instance with the same result. Node and NPM were updated to the latest version.

When I throw in sudo, it does the same thing locally, in cloud it doesn't even install and it gives me UNABLE_TO_GET_ISSUER_CERT_LOCALLY . I can install it locally (which is often recommended as a solution), and this works but when I try to "init" it, truffle complains that the folder is not empty.

Anybody has some idea how to get this resolved?

**Comment by cgewecke** [comment]
@Koldus This might be something `nvm` related. Did you follow all the instructions about bash profile aliasing [here](https://github.com/creationix/nvm#installation)? 

Are you able to install other packages globally and have them work as expected?

**Comment by Koldus** [comment]
I installed ganache-cli and everything went as it should.

**Comment by cgewecke** [comment]
Hmm. Not sure. I just tried this on my computer and it works. Are there any differences you can see that would help us isolate what the issue is? I'm on OSX, Sierra 
```
:~ cgewecke$ nvm list
         v8.9.4
->      v8.11.3
        v9.11.1
        v10.4.0
         system
default -> 8.9.4 (-> v8.9.4)
node -> stable (-> v10.4.0) (default)
stable -> 10.4 (-> v10.4.0) (default)
iojs -> N/A (default)
lts/* -> lts/carbon (-> v8.11.3)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.14.3 (-> N/A)
lts/carbon -> v8.11.3
:~ cgewecke$ nvm install lts/boron
Downloading and installing node v6.14.3...
Downloading https://nodejs.org/dist/v6.14.3/node-v6.14.3-darwin-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with shasum -a 256
Checksums matched!
Now using node v6.14.3 (npm v3.10.10)
:~ cgewecke$ npm install -g truffle
/Users/cgewecke/.nvm/versions/node/v6.14.3/bin/truffle -> /Users/cgewecke/.nvm/versions/node/v6.14.3/lib/node_modules/truffle/build/cli.bundled.js
/Users/cgewecke/.nvm/versions/node/v6.14.3/lib
└─┬ truffle@4.1.11 
  ├─┬ mocha@4.1.0 
  │ ├── browser-stdout@1.3.0 
  │ ├── commander@2.11.0 
  │ ├─┬ debug@3.1.0 
  │ │ └── ms@2.0.0 
  ... etc ...

:~ cgewecke$ truffle
Truffle v4.1.11 - a development framework for Ethereum

Usage: truffle <command> [options]

Commands:
  init      Initialize new and empty Ethereum project
  compile   Compile contract source files
  migrate   Run migrations to deploy contracts
  deploy    (alias for migrate)
  build     Execute build pipeline (if configuration present)
  
.. etc ..
```

**Comment by cgewecke** [comment]
@Koldus Any luck with this?

**Comment by Koldus** [comment]
@cgewecke Thanks a lot for the tips. I didn't get to it until today, but yes, I did manage to get it to work.

First, what helped was to initiate the truffle with npx.

I also tried to set up the complete environment from scratch. This time it worked perfectly. The only thing I did differently was that I also updated nvm to the latest version (not just node and npm). So that's probably what solved it for me.

Again, thanks a lot for the help.
